### PR TITLE
Fix Procedures UI

### PR DESCRIPTION
**Pull request by TheBrokenRail**

### CODE REVIEW

File: msg/messages.js [hunk 1]
```diff
@@ -1029,7 +1029,7 @@ Blockly.Msg.PROCEDURES_DEFNORETURN_HELPURL = 'https://en.wikipedia.org/wiki/Proc
 /// block text - This precedes the name of the function when defining it.  See
 /// [https://blockly-demo.appspot.com/static/apps/code/index.html?lang=en#c84aoc this sample
 /// function definition].
-Blockly.Msg.PROCEDURES_DEFNORETURN_TITLE = 'to';
+Blockly.Msg.PROCEDURES_DEFNORETURN_TITLE = 'Define';
```
Comment (dekrain) [inline msg/messages.js:1032]: Look [Here!](https://github.com/LLK/scratch-blocks/pull/595#issuecomment-247474885)

Comment (TheBrokenRail) [reply msg/messages.js:1032]: And Here: #607

Comment (dekrain) [reply msg/messages.js:1032]: But Scratch Team don't saw and don't approved that

**Comment by tmickel** [comment]
We'll be working on a redesign of this. The intention right now is only to have blocks to play with for scratch-vm development. So any effort on these is likely to be lost down the road, unfortunately.

**Pull request closed without merging**